services/build.gradle

```diff
@@ -13,6 +13,8 @@ description = "gRPC: Services"
     // Without this workaround, the project can not compile.
     it.options.compilerArgs += [
         "-Xlint:-deprecation",
```

Comment:
I think this can be removed here - it was there for some specific annoyances with fields deprecated in xds protos.

Reply:
Oh. I'm wrong. This was for reflection.proto